release notes/v0.46.0.md

```diff
@@ -0,0 +1,188 @@
+k6 `v0.46` is here 🎉! This release includes:
+
+- (_optional_) `<highlight of breaking changes>`
+- `<Summary of new features>` (_one or multiple bullets_)
+
```

Comment:
```suggestion
- xk6-browser extension version bump to `v1.0.2`, which includes breaking changes.
- Send custom headers to Loki log output
- Cloud Traces, the new integration between k6 and [Tempo](https://github.com/grafana/tempo)
- Ability to configure TLS per gRPC connection
- Cloud output v2
- And many UX improvements, bug fixes, and internal improvements
```

I tried writing up this first section here, but feel free to ignore it / make edits if it doesn't fit with how it was done in the previous release notes. 😅